venia-integration-tests/src/actions/routes/index.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
I dont think we need this any more. We moved these to custom Cypress actions in the wishlist PR.